service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/PlacementApplicationContentProvider.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableMap;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.service.DateFormatterService;
+import uk.gov.hmcts.reform.fpl.service.HearingBookingService;
+
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.CASE_TYPE;
+import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.JURISDICTION;
+import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.getFirstRespondentLastName;
+
+@Service
+public class PlacementApplicationContentProvider extends AbstractEmailContentProvider {
+    private final ObjectMapper mapper;
+
+    @Autowired
+    public PlacementApplicationContentProvider(@Value("${ccd.ui.base.url}") String uiBaseUrl,
+                                              ObjectMapper mapper,
+                                              HearingBookingService hearingBookingService,
+                                              DateFormatterService dateFormatterService) {
+        super(uiBaseUrl, dateFormatterService, hearingBookingService);
+        this.mapper = mapper;
+    }
+
+    public Map<String, Object> buildPlacementApplicationNotificationParameters(CaseDetails caseDetails) {
+        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
+
+        return ImmutableMap.of(
+            "respondentLastName", getFirstRespondentLastName(caseData.getRespondents1()),
+            "caseUrl", String.format("%1$s/case/%2$s/%3$s/%4$s",
```

Comment:
if you pull from master right now, there's a new method in EmailNotificationHelper called formatCaseUrl. Let's use that here, bit nicer

Reply:
the email template expects `CaseURL` but we are sending `caseUrl`. Could you confirm if its case-insensitive?